### PR TITLE
AppNotificationBuilder - Addressing last few feature nits

### PR DESCRIPTION
**Pull request by loneursid**
Addressing few nits from previous PRs

### CODE REVIEW

File: dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp [hunk 1]
```diff
@@ -367,27 +367,23 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
...
             GetImages().c_str(),
             GetProgressBars().c_str(),
-            L"</binding></visual>",
             m_audio.c_str(),
-            actions.c_str(),
-            L"</toast>") };
+            actions.c_str()) };
 
-        THROW_HR_IF_MSG(E_INVALIDARG, xmlResult.size() > c_maxAppNotificationPayload, "Maximum payload size exceeded");
+        THROW_HR_IF_MSG(E_FAIL, xmlResult.size() > c_maxAppNotificationPayload, "Maximum payload size exceeded");
```
Comment (loneursid) [inline dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp:386]: Not sure this checking the right size. What is the max payload size for an AppNotification? 5120 bytes? or 5120 unicode chars?

Comment (loneursid) [reply dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp:386]: @pmpurifoy - Any thoughts on this?

Comment (pmpurifoy) [reply dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp:386]: It's wchars.

Comment (loneursid) [reply dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp:386]: then looks like we're good.

thanks

**Review by pmpurifoy — APPROVED** [review]

**Pull request merged**